src/common/error/error-response-body.ts

```diff
@@ -17,7 +18,7 @@ export interface InternalErrorResponseBody {
     code?: number;
     detail?: string;
     instance?: string;
-    errors: string[] | { [key: string]: string };
```

Comment:
couldn't find an example of internal API returning `errors: { [key: string]: string };`